### ISSUE TITLE
ETA on code?

### ISSUE THREAD

**Issue by ghost** [vulkanprogrammingguide/examples#4]
@grahamsellers Do you have an estimate on when at least some of the code from your examples will be available? The book has been out since October 2016. It is a fantastic book and I've used it quite a bit, but being able to look at some of the code and shaders accompanying the listings and other writing in the book would be a huge help when tackling new parts of the API.

**Comment by SamuliRiihikoski** [comment]
Yeah. What's up with the code release. I have paid for the code too so if it's not going to release then it's refund for me please.
